### PR TITLE
chore: reorganize and update schemas

### PR DESCRIPTION
**Pull request by mcdurdin**
Now matches the layout of files in schemas in keymanapp/keyman. These ones are stored by version, unlike in the keyman repo.

This does not yet include the updated kmp.schema.json for 17.0. Will add that as a separate PR.

### CODE REVIEW

File: schemas/search/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# search
+
+* search.json
+
+Documentation at https://help.keyman.com/developer/cloud/search
+
+# search version history
+
+## 2019-12-13 1.0.2
```
Comment (mcdurdin) [inline schemas/search/README.md:9]: TODO: add 2.0 and 3.0 notes

File: schemas/keyman-touch-layout/2.1/keyman-touch-layout.clean.spec.json [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+        "id": { "$ref": "#/definitions/layer-id" },
+        "row": { "$ref": "#/definitions/rows" }
+      },
+      "required": ["id", "row"],
+      "additionalProperties": false
+    },
+
+    "layer-id": {
+      "type": "string",
+      "pattern": "^[a-zA-Z0-9_-]+$"
```
Comment (darcywong00) [inline schemas/keyman-touch-layout/2.1/keyman-touch-layout.clean.spec.json:50]: I was trying to find usage of an underscore layer ID in a keyman-touch-layouts file, when I saw 
[gff_amharic](https://raw.githubusercontent.com/keymanapp/keyboards/master/release/gff/gff_amharic/source/gff_amharic.keyman-touch-layout) had some non ascii characters in the layer ID
e.g. "id": "ዐ-layer",

Comment (mcdurdin) [reply schemas/keyman-touch-layout/2.1/keyman-touch-layout.clean.spec.json:50]: hmm. we may need to loosen this up, but that needs to be a separate issue to chase.

**Review by darcywong00 — APPROVED** [review]
lgtm

**Pull request merged**